cognite_toolkit/_cdf_tk/apps/_migrate_app.py

```diff
@@ -192,7 +202,9 @@ def source_system(
             data_set = ds_selector.select_data_set()
             dm_selector = DataModelingSelect(client, "migrate")
             instance_space = dm_selector.select_instance_space(
-                multiselect=False, message="In which instance space do you want to create the source system?"
+                multiselect=False,
+                message="In which instance space do you want to create the source system?",
+                include_emtpy=True,
```

Comment:
You typically migrate source-systems first, so the instance space is likely empty. Thus we need to include empty spaces.